activerecord/lib/active_record/enum.rb

```diff
@@ -75,6 +75,19 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
+  # You can use <tt>:prefix</tt> option then you need to define multiple enums
+  # with same values. If prefix is <tt>true</tt>, the methods are prefixed with
+  # the name of the enum.
+  #
+  #   class Invoice < ActiveRecord::Base
+  #     enum({verification: [:done, :fail]}, prefix: true)
```

Comment:
Those examples sound like they would be better off with suffixes to me. `visible_for_admin?` versus `admin_visible?`

@dhh you're better at making things read nicer. Do you have preference here?